zebra-rpc/src/methods/get_block_template_rpcs/types/submit_block.rs

```diff
@@ -1,12 +1,20 @@
+//! Parameter and response types for the `submitblock` RPC.
+
+// Allow doc links to these imports.
+#[allow(unused_imports)]
+use crate::methods::get_block_template_rpcs::GetBlockTemplateRpc;
+
 /// Optional argument `jsonparametersobject` for `submitblock` RPC request
 ///
-/// See notes for [`Rpc::submit_block`] method
+/// See notes for [`GetBlockTemplateRpc::submit_block`] method
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
 pub struct JsonParameters {
-    pub(crate) work_id: String,
+    pub(crate) work_id: Option<String>,
```

Comment:
The `work_id` field is also optional:
https://zcash.github.io/rpc/submitblock.html